idaes/models/properties/modular_properties/eos/tests/test_ceos_CO2_H2O.py

```diff
@@ -1,144 +0,0 @@
-#################################################################################
```

Comment:
Why is this file being removed?

Reply:
It is being removed as it does not belong here - this is a test depends on a file in `models_extra` and core tests should not depend on non-core models. I think I moved these tests somewhere else.